components/CurrentProjects.jsx

```diff
@@ -0,0 +1,56 @@
+import getFormattedDate from '../util/getFormattedDate';
+import Show from './Show';
+
+export default function CurrentProjects({ currentProjects }) {
+  return (
+    <section className="cmp-current-projects">
+      <Show when={!currentProjects.length}>
+        <h2 className="cmp-current-projects__header cmp-current-projects__header--heading">Current Projects</h2>
+        <p className="cmp-current-projects__status">This team member is not currently on a client project.</p>
+      </Show>
+
+      <Show when={currentProjects.length}>
+        <table className="cmp-current-projects__table">
+          <caption className="cmp-current-projects__header cmp-current-projects__header--caption">Current Projects</caption>
+          <thead className="cmp-current-projects__table-head">
+            <tr className="cmp-current-projects__table-header-row">
+              <th className="cmp-current-projects__table-header-cell">Project</th>
+              <th className="cmp-current-projects__table-header-cell">Client</th>
+              <th className="cmp-current-projects__table-header-cell">Start</th>
+              <th className="cmp-current-projects__table-header-cell">End</th>
+            </tr>
+          </thead>
+          <tbody className="cmp-current-projects__table-body">
+            {currentProjects.map((project) => {
+              const startDateString = getFormattedDate(project.startDate);
```

Comment:
Looks great! Nitpick ~ think about breaking this map into a variable for easier readability!